packages/end-user-strings/test/build-message-spec.ts

```diff
@@ -0,0 +1,10 @@
+import { expect } from "chai";
+import { buildMessage } from "../src/api";
+
+describe("buildMessage", () => {
+  it("will build an error message for the validation", () => {
+    const expectedErrorMessage = "Hello wonderful world";
+    const result = buildMessage("{0} wonderful {1}", "Hello", "world");
+    expect(expectedErrorMessage).to.be.equal(result);
```

Comment:
- `to.equal` not `to.be.equal` (or is it `to.eql` in chai?)
- Would this be clearer/readable: `expect(result).to.equal("Hello wonderful world");